pkg/controller/common/metrics.go

```diff
@@ -0,0 +1,67 @@
+package common
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// DefaultBindAddress is the port for the metrics listener
+	DefaultBindAddress = ":8797"
+
+	// MCCImportantConfigPaused logs when important config is being held up by pause
+	MCCImportantConfigPaused = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "mcc_important_config_paused",
+			Help: "important config type is waiting in specified paused pool",
+			// "platform config kubelet ca certificate is stuck in paused pool worker"
+		}, []string{"desc", "file", "pool"})
```

Comment:
Is there a benefit to the `desc` abbreviation over a full `description`?

You can also avoid landing the description in the metric itself by using path-to-description lookup logic in the alert's `description`.  I'm not familiar enough with Prom or the alert stack to know if that's worth the trouble.